R/praise.R

```diff
@@ -62,15 +6,21 @@
 #'
 #' @examples
 #' replicate(5, glue::glue("Random praise: {random_praise()}"))
-#' replicate(5, glue::glue("Random encouragement: {random_encourage()}"))
+#' replicate(5, glue::glue("Random encouragement: {random_encouragement()}"))
 #' @export
 #' @describeIn praise Returns a random praise message
 random_praise <- function() {
-  sample(.praise, 1)
+  sub("^Correct! ", "", learnr::random_praise())
```

Comment:
This worries me.

Can we instead update learnr::random_praise() to have a `pre` or `text` argument that defaults to `Correct! "`?  This way, we can supply `""` and not have to worry about it changing in the future.

Or even better... we remove `"Correct!"` entirely? Mark it as a breaking change. This would make it consistent with `random_encouragement()`.

Reply:
I'm in favor of removing it for consistency with `random_encouragement()`. I'll make a note in the corresponding learnr issue: [learnr#453](https://github.com/rstudio/learnr/issues/453)